packages/react/__tests__/__snapshots__/global.js.snap

```diff
@@ -18,25 +18,19 @@ exports[`basic 1`] = `
     data-emotion="css-global"
   >
     
-    @font-face{font-family:some-name;}
-  </style>
-  <style
-    data-emotion="css-global"
-  >
-    
-    @import url('something.com/file.css');
```

Comment:
have a pending fix in stylis for `@import` rules not being passed to the rulesheet plugin